packages/transformer/src/IModelImporter.ts

```diff
@@ -176,13 +166,19 @@ export class IModelImporter {
   }
 
   /**
-   * Marks an element so that it can be updated during import when [[IModelImportOptions.preserveElementIdsForFiltering]] is set to true
+   * Marks an element so that it can be updated during import when the [[IModelTransformOptions.preserveElementIdsForFiltering]] is set to true.
```

Comment:
```suggestion
   * Marks an element so that it can be updated during import when [[IModelTransformOptions.preserveElementIdsForFiltering]] is set to true.
```